packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,10 +1,5 @@
-# Apex Replay Debugger for Visual Studio Code (Beta)
-Apex Replay Debugger simulates a live debugging session using a debug log that is a recording of all interactions in a transaction. You no longer need to parse through thousands of log lines manually. Instead, Apex Replay Debugger presents the logged information similarly to an interactive debugger, so you can debug your Apex code. The debugging process is a repetition of editing your Apex code, pushing or deploying the code to your org, reproducing the buggy scenario, downloading the resulting debug log, and launching Apex Replay Debugger with that debug log.
```

Comment:
@ntotten - Was removing this introductory paragraph intentional? It's more informative than `This extension enables VS Code to use Apex Debug Logs to replay a debugging experience.` is.

Reply:
Opps, no. Fixed. Thanks!